### PR TITLE
Changed log message to warning as suggested in #828

### PR DESCRIPTION
**Pull request by rhnvrm**
Have changed the info logs as given in the email mentioned in #828

Done during PyconIndia 2016

### CODE REVIEW

**Comment by tmylk** [comment]
Looks good. Please add a line to CHANGELOG.md

**Comment by tmylk** [comment]
Please rebase to master to resolve the merge conflicts.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,9 @@ Changes
 =======
 
 0.13.2, 2016-08-19
-* export_phrases in Phrases model changed. Fixed issue #794 and added test cases in test/test_phrases.py(@AadityaJ,
-[#879](https://github.com/RaRe-Technologies/gensim/pull/879))
```
Comment (tmylk) [inline CHANGELOG.md]: Please don't delete other comments in changelog. 
Also please add this change to the future release 0.13.3

Comment (rhnvrm) [inline CHANGELOG.md]: Updated. Sorry for the bad commit. :+1:

**Comment by tmylk** [comment]
Thanks for the PR!

**Pull request merged**